CHANGELOG-1.1.md

```diff
@@ -0,0 +1,122 @@
+- Add new CRD TidbInitializer. ([#1391](https://github.com/pingcap/tidb-operator/pull/1391), [@aylei](https://github.com/aylei))
+- Upgrade the default backup image to pingcap/tidb-cloud-backup:20191217 and facilitate -r option ([#1360](https://github.com/pingcap/tidb-operator/pull/1360), [@aylei](https://github.com/aylei))
+- Terraform: fix docker ulimit configuring for latest EKS AMI. ([#1349](https://github.com/pingcap/tidb-operator/pull/1349), [@aylei](https://github.com/aylei))
+- Support sync pump status to tidb-cluster. ([#1292](https://github.com/pingcap/tidb-operator/pull/1292), [@shonge](https://github.com/shonge))
+- `tidb-controller-manager` will automatically create and reconcile the tidb-discovery-service now ([#1322](https://github.com/pingcap/tidb-operator/pull/1322), [@aylei](https://github.com/aylei))
+- make backup and restore more universal and secure ([#1276](https://github.com/pingcap/tidb-operator/pull/1276), [@onlymellb](https://github.com/onlymellb))
+- pd and tikv configs are now managed in TidbCluster resource. ([#1330](https://github.com/pingcap/tidb-operator/pull/1330), [@aylei](https://github.com/aylei))
+- The configuration of tidb-server can be managed in the `TidbCluster` resource now. ([#1291](https://github.com/pingcap/tidb-operator/pull/1291), [@aylei](https://github.com/aylei))
+- Add schema for configuration of TiKV. ([#1306](https://github.com/pingcap/tidb-operator/pull/1306), [@aylei](https://github.com/aylei))
+- tidb-initializer: wait for the TiDB host:port is open before processing to initialize TiDB. This speeds up TiDB initialization. ([#1296](https://github.com/pingcap/tidb-operator/pull/1296), [@cofyc](https://github.com/cofyc))
+- Remove DinD related scripts ([#1283](https://github.com/pingcap/tidb-operator/pull/1283), [@shonge](https://github.com/shonge))
+- backup chart: allow retrieving credentials from metadata on AWS and GCP ([#1248](https://github.com/pingcap/tidb-operator/pull/1248), [@gregwebs](https://github.com/gregwebs))
+- Add the privilege to operate configmap for tidb-controller-manager. ([#1275](https://github.com/pingcap/tidb-operator/pull/1275), [@aylei](https://github.com/aylei))
+- Manage TiDB service in tidb-controller-manager. ([#1242](https://github.com/pingcap/tidb-operator/pull/1242), [@aylei](https://github.com/aylei))
+- Support cluster-level setting for components ([#1193](https://github.com/pingcap/tidb-operator/pull/1193), [@aylei](https://github.com/aylei))
+- get time string from current time instead of pod name ([#1229](https://github.com/pingcap/tidb-operator/pull/1229), [@weekface](https://github.com/weekface))
+- Operator will not resign ddl owner when upgrading tidb-servers anymore because tidb-server will transfer ddl owner automatically on shutdown. ([#1239](https://github.com/pingcap/tidb-operator/pull/1239), [@aylei](https://github.com/aylei))
```

Comment:
```suggestion
- Operator will not resign the ddl owner anymore when upgrading tidb-servers because tidb-server will transfer ddl owner automatically on shutdown. ([#1239](https://github.com/pingcap/tidb-operator/pull/1239), [@aylei](https://github.com/aylei))
```

Reply:
What does "tidb-server will transfer ddl owner automatically on shutdown" mean？